Model/Order/OrderAddress.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace ScandiPWA\SalesGraphQl\Model\Order;
+
+use Magento\Sales\Api\Data\OrderAddressInterface;
+use Magento\Sales\Api\Data\OrderInterface;
+use Magento\SalesGraphQl\Model\Order\OrderAddress as CoreOrderAddress;
+
+/**
+ * Class to get the order address details
+ */
+class OrderAddress extends CoreOrderAddress
+{
+    /**
+     * Get the order Shipping address
+     *
+     * @param OrderInterface $order
+     * @return array|null
+     */
+    public function getOrderShippingAddress(
+        OrderInterface $order
+    ): ?array {
+        $shippingAddress = null;
+        if ($order->getShippingAddress()) {
+            $shippingAddress = $this->formatAddressData($order->getShippingAddress());
+        }
+        return $shippingAddress;
+    }
+
+    /**
+     * Get the order billing address
+     *
+     * @param OrderInterface $order
+     * @return array|null
+     */
+    public function getOrderBillingAddress(
+        OrderInterface $order
+    ): ?array {
+        $billingAddress = null;
+        if ($order->getBillingAddress()) {
+            $billingAddress = $this->formatAddressData($order->getBillingAddress());
+        }
+        return $billingAddress;
+    }
+
+    /**
+     * Customer Order address data formatter
+     *
+     * @param OrderAddressInterface $orderAddress
+     * @return array
+     */
+    private function formatAddressData(
```

Comment:
please don't make such methods private, this just blocks ability to plugin on them for no real reason

Imagine you had a custom column in an order that you wanted to add to the response.
Maybe there's a decent way to do it by plugging into the original method in the core that calls this method(I didn't check). 
But often the original method doesn't have the models such as $orderAddress as easily accessible arguments, and reloading them via a plugin can be bad enough performance that a preference to change private method becomes better.

Kust make it public unless a security risk